pkg/agent/agentclient/client.go

```diff
@@ -37,8 +37,8 @@ type AgentClient struct {
 }
 
 // NewAgentClient creates an AgentClient
-func NewAgentClient(address string, opts ...grpc.DialOption) (*AgentClient, error) {
-	stream, err := NewRedialableAgentClient(address, opts...)
+func NewAgentClient(address, saTokenFile string, opts ...grpc.DialOption) (*AgentClient, error) {
```

Comment:
nit:saTokenPath to be consisent with kubeconfigPath.

Reply:
renamed